### PR TITLE
PHP5-FPM / nginx: initial implementation

### PR DESCRIPTION
**Pull request by bryanlatten**
- [x] share values (not all) across php CLI and php FPM
- [x] invert static file check to simply _not_ be PHP file (since certain apps serve those contents dynamically)
- [x] nginx to log to stdout
- [x] create rewrite for php

### CODE REVIEW

File: container/root/run.sh [hunk 1]
```diff
@@ -1,20 +1,22 @@
 #!/bin/bash
 
+# As part of the "Two Phase" build, the first phase typically runs with composer keys mounted,
+# allowing the dependencies to be installed, the result of which is committed
 if [[ -f /root/.composer/config.json ]]
 then
   echo "Running `composer install`"
   composer install
-else
+  # Container exists after installation
```
Comment (jimdelois) [inline container/root/run.sh]: It... "exits", you mean?

**Comment by jimdelois** [comment]
:white_check_mark:

**Comment by chrishenry** [comment]
:+1:

**Comment by ingluisjimenez** [comment]
:+1:

**Pull request merged**